packages/reactackle-tabs/src/Tab/Tab.js

```diff
@@ -111,7 +93,7 @@ export default class Tab extends Component {
     if (!this.props.text) return null;
 
     return (
-      <TabTextStyled hasIcon={this.props.icon.name || this.props.icon.src}>
+      <TabTextStyled hasIcon={this.props.icon}>
```

Comment:
hasIcon has PropTypes.bool prop type => replace `this.props.icon` with `this.props.icon !== null`